subworkflows/local/prokka/subset/meta.yml

```diff
@@ -0,0 +1,48 @@
+name: "prokka_subset"
```

Comment:
Small suggestion for naming consistency: Either
- change this to `prokka_subsets` or
- change the workflow name in prokka/subset/main.nf from `PROKKA_SUBSETS` to `PROKKA_SUBSET`

I'd prefer the latter because the module name `prokka/subset` is singluar as well.